README.Ubuntu.bash

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-sudo apt-get update
-sudo apt-get install -y \
+apt-get update
```

Comment:
Why weren't these changes cherry-picked from commit d11b541?

Reply:
This commit contains other changes as well. I just took the whole file.

Reply:
Yes, you can cherry-pick only what you need, specifying the changes.